packages/storybook/.eslintrc.cjs

```diff
@@ -14,7 +14,22 @@ module.exports = {
                 'import/no-extraneous-dependencies': [
                     'error',
                     { devDependencies: true }
-                ]
+                ],
+                // This rule is disabled in order to allow linking directly to nimble-components and spright-components source
+                // Other files in the repo should NOT directly link to source and should resolve with the package identifier instead
+                'import/no-relative-packages': 'off',
+                'no-restricted-imports': [
```

Comment:
Added a lint rule to prevent folks from accidentally relying on both build output and original source files from the components packages